README.md

```diff
@@ -1 +1,20 @@
 # hw01
+# Introduction
+My name is Yilin Zhu, a graduate student from the MAPSS program majoring in sociology. This year I just finished my undergraduate education in sociology at Sun Yat-sen University.
+
+Here is a graduation photo taken with my best friend.
+<a href="https://sm.ms/image/k8R4mn1eZwzF5S7" target="_blank"><img src="https://i.loli.net/2020/10/04/k8R4mn1eZwzF5S7.jpg" ></a>
```

Comment:
there's a slight formatting issue here that when previewed, your image follows right after the last sentence rather than starting on a new line. I'm guessing it might have something to do with your using HTML code here rather than Markdown language. A Markdown code for the image should probably look like ![graduation_photo](https://i.loli.net/2020/10/04/k8R4mn1eZwzF5S7.jpg)

Reply:
lmaoo it is automatically displayed as a picture ...well you know what I mean here : ) if anything, just reference https://guides.github.com/features/mastering-markdown/

Reply:
Thank you for the instruction. It took me so long to figure out how to insert the photo.